### PR TITLE
Signup and login

### PR DESCRIPTION
**Pull request by breycarr**
All tests passing

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -1,3 +1,14 @@
-<form> {
-	align-items: center;
-}
\ No newline at end of file
+h1,h2 {
```
Comment (hughbric) [inline public/style.css]: We need to centralise our CSS. There is some CSS on the views HTML.

File: views/spaces/index.erb [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
+ <link rel="stylesheet" href="/style.css"/>
+</head>
 
-	div {
-		text-align: center;
-		margin-top: 50px;
-	}
-</style>
+<body>
+	<a href='/sessions/logout'>Sign out</a>
```
Comment (hughbric) [inline views/spaces/index.erb]: Indentation here seems inconsistent - 4 spaces instead of 2.

File: app.rb [hunk 3]
```diff
@@ -19,12 +20,37 @@ class MakersBNB < Sinatra::Base
...
+  get '/sessions/new' do
+    erb :login
+  end
+
+  post '/sessions/new' do
+    session[:user] = User.authenticate(
+      email: params[:email],
+      password: params[:password]
+    )
+    redirect '/sessions/new' if session[:user].nil?
```
Comment (hughbric) [inline app.rb]: Neat. Do these redirections tell the user why they're being redirected, or is this still a feature we're going to implement?

File: app.rb [hunk 4]
```diff
@@ -1,15 +1,16 @@
 require 'sinatra/base'
-require_relative './lib/user'
 require_relative 'lib/database_connection_setup'
 
 class MakersBNB < Sinatra::Base
   enable :sessions
```
Comment (hughbric) [inline app.rb]: Possible suggestion for the controller as a whole: grouping the routes according to feature with comment headings. Might improve readability.

File: app.rb (hunk 3 above)
Comment (breycarr) [reply app.rb]: That is a feature we're still yet to implement, I've added a checklist to Trello of error messages that we may want to implement going forward.

File: app.rb (hunk 4 above)
Comment (breycarr) [reply app.rb]: Great idea 💯

File: public/style.css (hunk 1 above)
Comment (breycarr) [reply public/style.css]: Agreed, another potential piece of learning could be how we create a standard header for elements that appear on most pages (the website title, logout button, etc)

File: views/spaces/index.erb (hunk 2 above)
Comment (breycarr) [reply views/spaces/index.erb]: Good spot, we'll change it!

File: public/style.css (hunk 1 above)
Comment (hughbric) [reply public/style.css]: Interesting.

**Pull request merged**